policyengine_us/variables/gov/states/nj/tax/income/credits/eitc/nj_childless_eitc_age_eligible.py

```diff
@@ -27,11 +25,11 @@ def formula(tax_unit, period, parameters):
         age = person("age", period)
         min_age = p.eligibility.age.min
         max_age = p.eligibility.age.max
-        meets_age_requirements = (age >= min_age) & (age <= max_age)
+        age_eligible = (age >= min_age) & (age <= max_age)
 
         return (
             ~separate
             & no_qualifying_children
-            & tax_unit.any(meets_age_requirements)
+            & tax_unit.any(age_eligible)
             & tax_unit("nj_eitc_income_eligible", period)
```

Comment:
if we're testing income as well as age, let's rename the variable to `nj_childless_eitc_eligible`